docs/features/search/README.md

```diff
@@ -0,0 +1,101 @@
+---
+id: search-overview
+title: Search Documentation
+sidebar_label: Overview
+description:
```

Comment:
insert `# prettier-ignore` between line 4 and 5, and make the description a oneliner - otherwise it will get cut off